src/bun.js/test/jest.zig

```diff
@@ -3592,6 +3592,202 @@ pub const Expect = struct {
         return .zero;
     }
 
+    pub fn toIncludeRepeated(this: *Expect, globalThis: *JSGlobalObject, callFrame: *CallFrame) callconv(.C) JSValue {
+        defer this.postMatch(globalThis);
+
+        const thisValue = callFrame.this();
+        const arguments_ = callFrame.arguments(2);
+        const arguments = arguments_.ptr[0..arguments_.len];
+
+        if (arguments.len < 2) {
+            globalThis.throwInvalidArguments("toIncludeRepeated() requires 2 arguments", .{});
+            return .zero;
+        }
+
+        const substring = arguments[0];
+        substring.ensureStillAlive();
+
+        if (!substring.isString()) {
+            globalThis.throw("toIncludeRepeated() requires the first argument to be a string", .{});
+            return .zero;
+        }
+
+        const count = arguments[1];
+        count.ensureStillAlive();
+
+        if (!count.isAnyInt()) {
+            globalThis.throw("toIncludeRepeated() requires the second argument to be a number", .{});
+            return .zero;
+        }
+
+        const countAsNum = std.fmt.parseInt(u32, count.toString(globalThis).toSlice(globalThis, default_allocator).slice(), 10) catch {
```

Comment:
We shouldn't use the zig std lib function for this. Instead, I think we can use .toNumber? One of the functions in JSValue will call `[Symbol.toPrimitive()]` and pass it a `"number"` which would do the coercions equivalent to `Number(string)` in JS.